edgeGw/jsonld-contexts/compound-edgeGateway.jsonld

```diff
@@ -1,7 +1,7 @@
 {
     "@context": [
         "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/edgeGw/jsonld-contexts/edgeGateway-context.jsonld",
-        "https://schema.lab.fiware.org/ld/context.jsonld",
+        "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/fiware/jsonld-contexts/labFiware-compound.jsonld",
```

Comment:
there is a core NGSI-LD context in this compound context, what will happen with the one just below?

Reply:
Not sure but I think it overwrites the one below.
May not add the whole compound context but just `https://fiware.github.io/data-models/context.jsonld`

Reply:
> Not sure but I think it overwrites the one below. May not add the whole compound context but just `https://fiware.github.io/data-models/context.jsonld`

Probably better.